package.json

```diff
@@ -15,7 +15,7 @@
     "fuse.js": "^3.2.0",
     "intl": "1.2.5",
     "jail-monkey": "0.1.0",
-    "mattermost-redux": "mattermost/mattermost-redux",
+    "mattermost-redux": "mattermost/mattermost-redux#b9b0208da40407fb3c77875117e0687552466f02",
```

Comment:
we shouldn't be including the commit here, only in the yarn.lock file

Reply:
Why not? This is how we do it in the web app

Reply:
There is no need as yarn.lock already takes care of it so is redundant unless we plan on using npm, actually in the webapp shouldn't be necessary either

Reply:
That's a good point. I've been either manually updating the hashes which is error prone or using `yarn add mattermost-redux#....` which kind of messes up the yarn.lock. I'll revert this line